### PR TITLE
Mind your Dos and Donts

### PR DESCRIPTION
**Pull request by jaketrent**
asdf

### CODE REVIEW

File: docs/src/components/guideline.module.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.guideline {
```
Comment (danethurber) [inline docs/src/components/guideline.module.css:1]: i think some bottom margin would help 

Screen Shot 2020-09-14 at 1 43 50 PM

**Review by danethurber — CHANGES_REQUESTED** [review]
good to merge after you look at my one comment

also there is no dark theme change - @sweeting any thoughts on that?

File: docs/src/components/guideline.module.css (hunk 1 above)
Comment (jaketrent) [reply docs/src/components/guideline.module.css:1]: mmm, yes, always helps

**Review by danethurber — APPROVED** [review]

**Comment by jaketrent** [comment]
add lots o margin and dark theme suPPort

**Pull request merged**